docs/ide/quickstart-nodejs.md

```diff
@@ -15,94 +15,115 @@ dev_langs:
+   - Highlighted in bold is your project, using the name provided when you set up the project. On disk, this project is represented by a *.njsproj* file in your project folder.
 
    - At the top level is a solution, which by default has the same name as your project. A solution, represented by a *.sln* file on disk, is a container for one or more related projects.
 
-   - The npm node shows any installed npm packages. You can right-click the npm node to search for and install npm packages using a dialog box.
+   - The **npm** node shows installed npm packages. You can right-click the npm node to search for and install npm packages using a dialog box.
 
 1. If you want to install npm packages or Node.js commands from a command prompt, right-click the project node and choose **Open Command Prompt Here**.
 
    ![Node.js command prompt](../ide/media/quickstart-nodejs-command-prompt.png)
 
-1. In the *server.js* file in the editor (left pane), choose `http.createServer` and then press **F12** or choose **Go To Definition** from the context (right-click) menu. This command takes you to the definition of the `createServer` function in *index.d.ts*.
+1. In the open *server.js* file, select `http.createServer` and press **F12** or choose **Go To Definition** from the context (right-click) menu. This command takes you to the definition of the `createServer` function in *index.d.ts*.
 
    ![Go To Definition context menu](../ide/media/quickstart-nodejs-gotodefinition.png)
 
-1. Got back to *server.js*, then put your cursor at the end of the string in this line of code, `res.end('Hello World\n');`, and modify it so that it looks like this:
+1. Got back to *server.js* and locate this line of code: `res.end('Hello World\n');`. Modify the code like this:
 
     `res.end('Hello World\n' + res.connection.`
 
-    Where you type `connection.`, IntelliSense provides options to auto-complete the code entry.
+    When you type `connection.`, IntelliSense provides options to auto-complete the code entry.
 
    ![IntelliSense auto-complete](../ide/media/quickstart-nodejs-intellisense.png)
 
-1. Choose **localPort**, and then type `);` to complete the statement so that it looks like this:
+1. Choose **localPort** and type `);` to complete the statement:
 
     `res.end('Hello World\n' + res.connection.localPort);`
 
-## Run the application
-
-1. Press **Ctrl**+**F5** (or **Debug > Start Without Debugging**) to run the application. The app opens in a browser.
+## Run the app
 
-1. In the browser window, you will see "Hello World" plus the local port number.
+1. Press **Ctrl**+**F5** (or **Debug > Start Without Debugging**) to run the app. 
+ 
+   The app opens in a browser.
 
-1. Close the web browser.
+1. In the browser, verify that you see a "Hello World" message and the local port number.
```

Comment:
Should we provide more detail about what port number the user should expect to see?

Reply:
I think it's OK as-is. It's not something that will change unless they change it explicitly.